### PR TITLE
Presets for ProfilerOptions

### PR DESCRIPTION
**Pull request by lovleen3112**
Added the following presets:
1. complete
2. data_types
3. numeric_stats_disabled

The test cases are also added in dataprofiler/tests/profilers/profiler_options/test_profiler_preset.py

### CODE REVIEW

File: dataprofiler/tests/profilers/profiler_options/test_profiler_preset.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+@mock.patch(
+    "dataprofiler.profilers.data_labeler_column_profile." "DataLabelerColumn.update",
+    return_value=None,
+)
+@mock.patch("dataprofiler.profilers.profile_builder.DataLabeler", spec=BaseDataLabeler)
+class TestProfilerPreset(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.data = Data(data=pd.DataFrame([1, 2]), data_type="csv")
```
Comment (JGSweets) [inline dataprofiler/tests/profilers/profiler_options/test_profiler_preset.py:21]: I think we can remove this since we aren't using it. Otherwise LGTM

**Comment by JGSweets** [comment]
FYI, @pietz 
https://github.com/capitalone/DataProfiler/issues/604

**Review by JGSweets — APPROVED** [review]

**Review by taylorfturner — APPROVED** [review]
Beautiful! LGTM

**Review by micdavis — APPROVED** [review]

**Comment by JGSweets** [comment]
Closing PR as coming in #638  due to CLA issues

**Comment by pietz** [comment]
Wow awesome!

**Comment by taylorfturner** [comment]
@pietz #638 is where the feature was actually merged to `main`.

**Pull request closed without merging**